### ISSUE TITLE
Need more clarification on format update as it can have accessibility issues

### ISSUE THREAD

**Issue by ssanupam24** [dlibby-/EditContext#11]
There have been discussions (can't find the bug number from the chromium database) on whether IME should be allowed to apply its own styles in an edit context or not. These formats can affect accessibility so it is important that we clarify this part in the design. Ex: A blue highlight for IME reconversion may not be what the user wants to see on a webpage that has a blue background color for some reason.
